src/python/pants/engine/scheduler.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+
```

Comment:
Nit: not sure if this was intentional, but if so I don't think it's a good change. Generally, we use one space for imports.

Feel free to ignore if you don't end up making any other changes.